app/utils/bundle-importer.js

```diff
@@ -460,7 +481,20 @@ YUI.add('bundle-importer', function(Y) {
               name = requiredModel.get('name') + '/' + size;
               break;
             case 2:
-              record.args[2] = requiredModel.id;
+              // Check whether this placement refers to a machine or a unit.
+              if (arg.indexOf('$addUnit') === 0) {
```

Comment:
I try to use `>-1` because then it's clear that you meant "if it exists" and it wasn't a typo meaning doesn't exist (0)